jetpack.php

```diff
@@ -12,6 +12,10 @@
  * Domain Path: /languages/
  */
 
+if ( ! defined( 'VIP_JETPACK_DEFAULT_VERSION' ) ) {
+	define( 'VIP_JETPACK_DEFAULT_VERSION', '8.9' );
```

Comment:
As there won't exist a `jetpack` directory, we need to set the default version here. This will need to be updated on every upgrade to a new minor version.